src/html/sandbox.html

```diff
@@ -65,27 +65,66 @@
                     return '';
                 }
             })
-
             window.addEventListener('message', function (event) {
                 var command = event.data.command;
-                if (command === "render") {
-                    event.source.postMessage({
-                        html: htmlTemplate(event.data.context),
-                        callback: event.data.callback,
-                        openTemplate: event.data.openTemplate
-                    }, event.origin);
+                switch(command) {
+                    case "renderTemplates":
+                        var source = document.getElementById('edit_template').innerHTML;
+                        var htmlTemplate = Handlebars.compile(source);
+                        event.source.postMessage({
+                            content: 'template-editor',
+                            html: htmlTemplate(event.data.context),
+                            callback: event.data.callback,
+                            openTemplate: event.data.openTemplate
+                        }, event.origin);
+                        break;
+                    case "renderDomain":
+                        var domainSource = document.getElementById('addDomains').innerHTML;
+                        var htmlDomainTemplate = Handlebars.compile(domainSource);
+                            event.source.postMessage({
+                            content: 'domain-list',
+                            html: htmlDomainTemplate(event.data.context),
+                            callback: event.data.callback
+                        }, event.origin);
+                        break;
                 }
             });
         </script>
+
+        <script id="addDomains" type="text/x-handlebars-template">
+            {{#each domains}}
+                {{#if this.default}}
+                    <li class='collection-item container custom-domain-collection'>
+                        <div class='row valign-wrapper remove-margin'>
+                            <div class='col s10 left-align custom-domain-label'>
+                                {{ this.name }} 
+                            </div>
+                            <div class='col s2'>
+                            </div>
+                        </div>
+                    </li>
+                {{else}}
+                    <li class='collection-item custom-domain-collection container'>
+                        <div class='row valign-wrapper remove-margin' style: >
+                            <div class="col s10 left-align custom-domain-label">
+                                {{ this.name }} 
+                            </div>
+                            <div class='col s2'>
+                                <a class='btn-floating btn-jti tooltipped' data-position='left' data-delay='50' data-tooltip='Remove Domain'> 
+                                    <i id= {{this.id}} class='customDomainRemoveButton material-icons'>delete</i> 
```

Comment:
Spacing around `id`, and class names have been using `-` notation, not camelCase

Reply:
:+1: